app/models/user.rb

```diff
@@ -131,6 +131,8 @@ def can_be_invited_through_phone_or_email?
 
   def soft_delete
     update_columns(
+      last_name: "Usager supprimé",
+      first_name: "Usager supprimé",
```

Comment:
J'aime bien ce que fait rdv-sp où ils mettent des crochets pour différencier visuellement ce qui a été fait par un robot et ce qui a été rentré par un usager
```suggestion
      last_name: "[Usager supprimé]",
      first_name: "[Usager supprimé]",
```